recipes/universal-ctags/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "universal-ctags" %}
+{% set version = "5.9.20201122.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/universal-ctags/ctags/archive/p{{ version }}.tar.gz
+  sha256: dda5d86cef6aeaef89f707292e49a8603be037053381dd378b60a560f7a241f6
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pkg-config
+    - make
+  host:
+    - libiconv
+    - libxml2
+    - jansson
+    - libseccomp  # [linux]
```

Comment:
Why?

Reply:
Meaning, why not on Linux?

Reply:
This is marked as Linux-only as it's a Linux-only package.

Reply:
@mbargull Are you reading the condition in the wrong way?

Reply:
What do you mean? I've never said anything ;). Bye!

Reply:
(Yep, sorry for the noise!)